troubleshooting/tools/color-logger/README.md

```diff
@@ -0,0 +1,48 @@
+The logger will create 3 types of log events.
+
+
+1. Type 1: controlled by PAYLOAD_COUNT
+
+```
+{
+    "time": # timestamp,
+    "gauge": # a counter from 0 to PAYLOAD_COUNT - 1,
+    "uuid": "abc{gauge}",
+    "payload": "x" * PAYLOAD_SIZE
+}
+```
+
+You can search for specific events easily in CW Logs UI with
+the value of the "uuid" field. For example, search "abc77".
+
+2. Type 2: Events sent once for every 100 Type 1 events sent
+
+```
+{
+    "blue": True,
+    "id": # a counter from 0 to PAYLOAD_COUNT - 1,
+    "time": # timestamp,
+}
+```
+
+Search for "blue" in CW Logs UI and check for id's 0, 100, 200, 300... 
+
+3. Type 3: sent once for every 71 Type 1 events, up to SIGNAL_PAYLOAD_COUNT
+
+```
+{
+    "payload": "x" * SIGNAL_PAYLOAD_SIZE,
+    "event_id": event_id,
+    "counter": # ID from 0 to SIGNAL_PAYLOAD_COUNT,
+    "global_counter": # same as ID in type 2 above, allows you to check these large logs 
+    # with the smaller ones that are sent before and after them
+    "time": # timestamp,
+}
```

Comment:
This to signal a cloudwatch flush by dumping a bunch of bytes so the threshold for flushing is crossed? Not clear why 71, a prime number is chosen.

Reply:
I think I somewhat randomly chose it... I think I was thinking a prime number would be best since lots of bugs might affect events at multiples of nicer numbers... so this adds some randomness


this logger wasn't completely deeply thought out... it could definitely be improved

Reply:
Makes sense to me!